src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+
```

Comment:
Header comment is missing. A good description is `Represents an adding or removing a Tag to/from a Person`.